.github/workflows/client.yml

```diff
@@ -34,3 +33,28 @@ jobs:
       - run: cd client/ionic && npm install
       - run: cd client/ionic && npm audit --audit-level=moderate
       - run: cd client/ionic && CI=true npm test
+  # This builds the android APK
+  build-android-debug:
+    strategy:
+      matrix:
+        platform: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '14'
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '10.x'
+      - run: npm install -g @ionic/cli
+      - run: npm install
+        working-directory: ./client/ionic
+      - run: ionic build
+        working-directory: ./client/ionic
+      - run: ionic cap sync
+        working-directory: ./client/ionic
+      - run: ./gradlew test
+        working-directory: ./client/ionic/android
+      - run: ./gradlew build
+        working-directory: ./client/ionic/android
```

Comment:
nit: EOF newline

Reply:
simplest fix for this going forward is Editorconfig file if we want to add